halo2_backend/src/helpers.rs

```diff
@@ -105,7 +105,8 @@ impl<F: PrimeField + SerdeObject> SerdePrimeField for F {}
 /// Convert a slice of `bool` into a `u8`.
 ///
 /// Panics if the slice has length greater than 8.
-pub fn pack(bits: &[bool]) -> u8 {
+#[allow(dead_code)]
+pub(crate) fn pack(bits: &[bool]) -> u8 {
```

Comment:
If this is not `pub` but needs `#[allow(dead_code)]` I think it means that this function is not used internally, but also not exposed publicly.  So maybe it's better to remove it?

I think this was only used here https://github.com/privacy-scaling-explorations/halo2/blob/73408a140737d8336490452193b21f5a7a94e7de/halo2_proofs/src/plonk.rs#L100

The same applies to `unpack`.

Reply:
Exactly.  Actually I'm not sure why this was there before.  Previously, for fixed columns only the commitment was stored; and for selectors it was the assignment that was stored.

Reply:
Done https://github.com/privacy-scaling-explorations/halo2/pull/307/commits/dcb667a4e4b1e168fa9fb10b77405bf1edd75d04